holidays/countries/israel.py

```diff
@@ -128,35 +146,35 @@ def _populate_school_holidays(self):
         if self._year >= 2101:
             raise NotImplementedError
 
-        # Sukkot Chol HaMoed
-        sukkot_dt = self._get_holiday("SUKKOT")
+        sukkot_dt = self._get_holiday(SUKKOT)
         for offset in range(1, 6):
-            self._add_holiday("Sukkot", sukkot_dt + td(days=offset))
+            # Chol HaMoed Sukkot (Feast of Tabernacles holiday).
+            self._add_holiday(tr("חול המועד סוכות"), sukkot_dt + td(days=offset))
 
-        # Hanukkah
-        for yr in (self._year - 1, self._year):
-            hanukkah_dt = _HebrewLunisolar.hebrew_holiday_date(yr, "HANUKKAH")
+        for year in (self._year - 1, self._year):
+            hanukkah_dt = _HebrewLunisolar._get_holiday(HANUKKAH, year)
             for offset in range(8):
-                self._add_holiday("Hanukkah", hanukkah_dt + td(days=offset))
+                # Hanukkah.
+                self._add_holiday(tr("חנוכה"), hanukkah_dt + td(days=offset))
 
-        # Fast of Esther
-        name = "Fast of Esther"
-        purim_dt = self._get_holiday("PURIM")
+        # Ta`anit Ester (Fast of Esther).
+        name = tr("תענית אסתר")
+        purim_dt = self._get_holiday(PURIM)
         taanit_ester_dt = purim_dt + td(days=-1)
-        is_obs, _ = self._add_observed(taanit_ester_dt, name, SAT_TO_PREV_THU)
-        if not is_obs:
+        is_observed, _ = self._add_observed(taanit_ester_dt, name, SAT_TO_PREV_THU)
+        if not is_observed:
             self._add_holiday(name, taanit_ester_dt)
 
-        # Purim
-        self._add_holiday("Purim", purim_dt)
+        # Purim.
+        self._add_holiday(tr("פורים"), purim_dt)
 
-        # Passover (Chol HaMoed Pesach)
-        passover_dt = self._get_holiday("PASSOVER")
+        passover_dt = self._get_holiday(PASSOVER)
         for offset in range(1, 6):
-            self._add_holiday("Passover", passover_dt + td(days=offset))
+            # Chol HaMoed Pesach (Passover holiday).
+            self._add_holiday(tr("חול המועד פסח"), passover_dt + td(days=offset))
```

Comment:
for organization, move this up to be next to `passover_dt`.

Reply:
That's a different category (public vs school holidays). I'd definitely have them combined if they were same kind.